apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -361,26 +361,6 @@ type EnvoyListenerConfig struct {
 	TLS *EnvoyTLS `json:"tls,omitempty"`
 }
 
-// TLSCipherType is a string alias for the TLS ciphers supported by Envoy.
-//
-// Values:
-//
-// * [ECDHE-ECDSA-AES128-GCM-SHA256|ECDHE-ECDSA-CHACHA20-POLY1305]
-// * [ECDHE-RSA-AES128-GCM-SHA256|ECDHE-RSA-CHACHA20-POLY1305]
-// * ECDHE-ECDSA-AES128-GCM-SHA256
-// * ECDHE-RSA-AES128-GCM-SHA256
-// * ECDHE-ECDSA-AES128-SHA
-// * ECDHE-RSA-AES128-SHA
-// * AES128-GCM-SHA256
-// * AES128-SHA
-// * ECDHE-ECDSA-AES256-GCM-SHA384
-// * ECDHE-RSA-AES256-GCM-SHA384
-// * ECDHE-ECDSA-AES256-SHA
-// * ECDHE-RSA-AES256-SHA
-// * AES256-GCM-SHA384
-// * AES256-SHA
-type TLSCipherType string
```

Comment:
removed this type since it didnt seem to be getting us anything, string list seems like it should be just fine to use